app/src/main/java/org/andsopt/android/yes24ticket/presentation/ui/component/FloatingButton.kt

```diff
@@ -0,0 +1,78 @@
+package org.andsopt.android.yes24ticket.presentation.ui.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import org.andsopt.android.yes24ticket.R
+import org.andsopt.android.yes24ticket.ui.theme.Yes24TicketTheme
+
+@Composable
+fun ScrollToTopFloatingButton(
+    onClick: () -> Unit,
+    sizeDp: Dp = 30.dp,
+    modifier: Modifier = Modifier,
```

Comment:
```suggestion
    modifier: Modifier = Modifier,
    sizeDp: Dp = 30.dp,
```
Modifier 위치는 필수 파라미터 (여기서는 onClick) 바로 아래에 둬주세요 !

Reply:
넵!